apps/minifront/src/state/swap/index.ts

```diff
@@ -31,9 +32,9 @@ interface Actions {
   setBalancesResponses: (balancesResponses: BalancesResponse[]) => void;
   setStakingAssetMetadata: (metadata: Metadata) => void;
   setSwappableAssets: (assets: Metadata[]) => void;
-  setAssetIn: (asset: BalancesResponse) => void;
+  setAssetIn: (asset?: BalancesResponse) => void;
   setAmount: (amount: string) => void;
-  setAssetOut: (metadata: Metadata) => void;
+  setAssetOut: (metadata?: Metadata) => void;
```

Comment:
Are these changes necessary? Won't they always be defined if these methods are being called?

Reply:
Ah, I see why they're needed from the swap loader.

I have a PR (#1316) to remove this loader, so I guess it's up to you whether to rebase on that once it's merged and then change this requirement, or just change this requirement now and tweak the loader which is going to be removed soon anyway... if that makes sense 😆

Reply:
Noticed the removal of swap loader. I added some changes to the zquery fetchers of swappableAssets and balancesResponses, so this code with optional parameters is irrelevant now